From 8fbe73cbbf524cfa6955a10a6617f02161ffa33b Mon Sep 17 00:00:00 2001
From: Wei-Ning Huang <w@cobinhood.com>
Date: Fri, 21 Sep 2018 18:12:20 +0800
Subject: [PATCH] common.mk: remove hard coded CFLAGS and LDFLAGS

---
 common.mk | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/common.mk b/common.mk
index 3a82835..9b42060 100644
--- a/common.mk
+++ b/common.mk
@@ -14,9 +14,6 @@ ifeq ($(UNAME_S),Darwin)
   OS=mac
   ARCH=x86_64
   LIB_SUF=dylib
-  OPENSSL_DIR?=/usr/local/opt/openssl
-  CFLAGS+=-I$(OPENSSL_DIR)/include
-  LDFLAGS+=-L$(OPENSSL_DIR)/lib
 else
   LIB_SUF=so
 endif
-- 
2.17.1 (Apple Git-112)

